core/app/[locale]/(default)/account/[tab]/_components/wishlist-sheet/wishlist-sheet-content.tsx

```diff
@@ -0,0 +1,84 @@
+import { useTranslations } from 'next-intl';
+import { Dispatch, SetStateAction, useState } from 'react';
+
+import { AccountStatusProvider } from '~/app/[locale]/(default)/account/[tab]/_components/account-status-provider';
+import { CreateWishlistForm } from '~/app/[locale]/(default)/account/[tab]/_components/create-wishlist-form';
+import { Modal } from '~/app/[locale]/(default)/account/[tab]/_components/modal';
+import { createWishlist } from '~/client/mutations/create-wishlist';
+
+import { Button } from '../../../../../../../components/ui/button';
+
+import { UpdateWishlistsForm } from './update-wishlists-form';
+
+import { Wishlist } from '.';
+
+export type NewWishlist = NonNullable<Awaited<ReturnType<typeof createWishlist>>>;
+
+interface WishlistSheetContentProps {
+  productId: number;
+  setWishlists: Dispatch<SetStateAction<Wishlist[]>>;
+  wishlists: Wishlist[];
+}
+
+export const WishlistSheetContent = ({
+  productId,
+  setWishlists,
+  wishlists,
+}: WishlistSheetContentProps) => {
+  const t = useTranslations('Account.Wishlist.Sheet');
+
+  const [ceateWishlistModalOpen, setCreateWishlistModalOpen] = useState(false);
+
+  const handleWishlistCreated = (newWishlist: NewWishlist) => {
+    setCreateWishlistModalOpen(false);
+
+    const wishlist = {
+      ...newWishlist,
+      items: [],
+    };
+
+    setWishlists((prevWishlists) => {
+      const newWishlists = [...prevWishlists, wishlist];
+
+      return newWishlists;
+    });
```

Comment:
since we don't reuse wishlist variable we can simplify even further🤔

```
setWishlists((prevWishlists) => ([...prevWishlists, {
  ...newWishlist,
  items: [],
}]));
```

Reply:
just forgot to get rig of this var after consoling 💅